### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by gaetanars**
This PR to add little improvments on documentation.

We build a Crossplane provider for FlexibleEngine on top of this terraform Provider and we need this improvment on our side to do better build with Upjet Framework.

Thanks

### CODE REVIEW

File: docs/resources/apig_api.md [hunk 1]
```diff
@@ -69,9 +71,9 @@ The following arguments are supported:
 
 * `request_path` - (Required, String) Specifies the request address, which can contain a maximum of 512 characters
   request parameters enclosed with brackets ({}).
-  + The address can contain special characters, such as asterisks (), percent signs (%), hyphens (-), and
+  * The address can contain special characters, such as asterisks (), percent signs (%), hyphens (-), and
```
Comment (ShiChangkuo) [inline docs/resources/apig_api.md:74]: [MD004 - Unordered list style](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md004---unordered-list-style) requires that each sublist has a consistent symbol that differs from its parent list.

**Comment by ShiChangkuo** [comment]
@gaetanars I will add a workflow with markdownlint and fix the previous errors.

File: docs/resources/antiddos_v1.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ---
 subcategory: "Anti-DDoS"
+description: ""
+page_title: "flexibleengine_antiddos_v1"
```
[the remark this answers is not in the document]
Comment (gaetanars) [reply docs/resources/antiddos_v1.md:4]: You're right, but when we check the [terraform-plugin-docs](https://github.com/hashicorp/terraform-plugin-docs) from hashicorp it produces this rendering.
To build the FlexibleEngine Crossplane  provider, Upjet framework works with Terraform documentation and needs this attributes to work.

File: docs/resources/apig_api.md (hunk 1 above)
Comment (azrod) [reply docs/resources/apig_api.md:74]: I agree on the understanding of the MD004 error but in my IDE or on the [demo site](https://dlaa.me/markdownlint/) the use of a `+` generates an error. See screenshot. 

Capture d’écran 2022-12-15 à 09 11 24

Capture d’écran 2022-12-15 à 09 20 31

Comment (ShiChangkuo) [reply docs/resources/apig_api.md:74]: you should config the style of MD004 to "sublist"

```json
{
    "MD004": {
        "style": "sublist"
    }
}
```

**Comment by gaetanars** [comment]
@ShiChangkuo I have cleaned ours commits and resolve conflicts with your recent improvements.

**Comment by ShiChangkuo** [comment]
@gaetanars I suggest that this PR contains two commits for docs:
1. remove *\\* character.
2. Add more frontmatter attributes;

**Comment by gaetanars** [comment]
@ShiChangkuo Ok, now this PR is more clean :)

File: docs/resources/vpc_route.md [hunk 3]
```diff
@@ -93,6 +95,6 @@ This resource provides the following timeouts configuration options:
 
 VPC routes can be imported using the route table ID and their `destination` separated by a slash, e.g.
 
-```
-$ terraform import flexibleengine_vpc_route.test &ltroute_table_id&gt/&ltdestination&gt
+```shell
+terraform import flexibleengine_vpc_route.test &ltroute_table_id&gt/&ltdestination&gt
```
Comment (ShiChangkuo) [inline docs/resources/vpc_route.md:99]:
```suggestion
terraform import flexibleengine_vpc_route.test <route_table_id>/<destination>
```

Comment (gaetanars) [reply docs/resources/vpc_route.md]: It's fixed, I have found 3 files with this typo.

**Pull request merged**